ci/builders/README.md

```diff
@@ -387,15 +410,14 @@ downloaded. This section explains how to triage a local generator.
 
 ### Prerequisites (one time installation)
 
-
 #### Install CAS utility
 
 CAS client is required to download the sub-build artifacts. To install
```

Comment:
What are the use-cases for this? Why would someone want to do this?

Reply:
Added a paragraph with an example.